<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkDICOMBrowser</class>
 <widget class="QWidget" name="ctkDICOMBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DICOM Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="ToolBar">
     <property name="windowTitle">
      <string>DICOM Browser Toolbar</string>
     </property>
     <property name="floatable">
      <bool>true</bool>
     </property>
     <addaction name="ActionImport"/>
     <addaction name="ActionExport"/>
     <addaction name="ActionQuery"/>
     <addaction name="ActionSend"/>
     <addaction name="ActionRemove"/>
     <addaction name="ActionRepair"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="DatabaseDirectoryProblemFrame">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);background-color: rgb(245, 245, 170);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="DatabaseDirectoryProblemLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Warning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="UpdateDatabaseButton">
        <property name="text">
         <string>Update database</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CreateNewDatabaseButton">
        <property name="text">
         <string>Create new database</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SelectDatabaseDirectoryButton">
        <property name="text">
         <string>Select database folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="QueryLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ctkDICOMTableManager" name="dicomTableManager">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="dynamicTableLayout">
        <bool>true</bool>
       </property>
       <property name="m_DynamicLayout" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="InformationMessageFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="InformationMessageLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InformationMessageDismissButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ProgressFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QProgressBar" name="ProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ProgressCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ProgressLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Progress</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="ProgressDetailLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>300</number>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkDirectoryButton" name="DirectoryButton"/>
   </item>
  </layout>
  <action name="ActionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import a DICOM file or folder</string>
   </property>
  </action>
  <action name="ActionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export selected series to a DICOM folder</string>
   </property>
  </action>
  <action name="ActionQuery">
   <property name="text">
    <string>Query</string>
   </property>
   <property name="toolTip">
    <string>Query and retrieve studies from a DICOM server</string>
   </property>
  </action>
  <action name="ActionSend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>Send selected series to DICOM server</string>
   </property>
  </action>
  <action name="ActionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected series from database</string>
   </property>
  </action>
  <action name="ActionRepair">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repair</string>
   </property>
   <property name="toolTip">
    <string>Check whether all the files associated with images in the local Database are available on the disk.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMTableManager</class>
   <extends>QWidget</extends>
   <header>ctkDICOMTableManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
   <container>1</container>
   <slots>
    <signal>directoryChanged(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ActionImport</sender>
   <signal>triggered(bool)</signal>
   <receiver>ctkDICOMBrowser</receiver>
   <slot>openImportDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActionExport</sender>
   <signal>triggered(bool)</signal>
   <receiver>ctkDICOMBrowser</receiver>
   <slot>openExportDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActionQuery</sender>
   <signal>triggered(bool)</signal>
   <receiver>ctkDICOMBrowser</receiver>
   <slot>openQueryDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActionRemove</sender>
   <signal>triggered()</signal>
   <receiver>ctkDICOMBrowser</receiver>
   <slot>onRemoveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActionRepair</sender>
   <signal>triggered()</signal>
   <receiver>ctkDICOMBrowser</receiver>
   <slot>onRepairAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActionSend</sender>
   <signal>triggered()</signal>
   <receiver>ctkDICOMBrowser</receiver>
   <slot>openSendDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImportDialog()</slot>
  <slot>openExportDialog()</slot>
  <slot>openQueryDialog()</slot>
  <slot>openSendDialog()</slot>
  <slot>onDatabaseDirectoryChaged(QString)</slot>
  <slot>onNextImage()</slot>
  <slot>onPreviousImage()</slot>
  <slot>onNextSeries()</slot>
  <slot>onPreviousSeries()</slot>
  <slot>onNextStudy()</slot>
  <slot>onPreviousStudy()</slot>
  <slot>onAutoPlayCheckboxStateChanged(int)</slot>
  <slot>onThumbnailWidthSliderValueChanged(int)</slot>
  <slot>onRemoveAction()</slot>
 </slots>
</ui>
